app/models/loan.rb

```diff
@@ -97,7 +97,7 @@ def recalculate_loan_health
   def default_name
     if organization
       date = signing_date || created_at.to_date
-      "#{organization.name} - #{I18n.l(date)}"
+      "#{organization.name} - #{ApplicationController.helpers.ldate(date)}"
```

Comment:
Nice job getting this to work but we definitely want to avoid coupling the model to a view helper. This is not a good dependency.

Ideally we would not be doing this kind of work in the model at all, but in a decorator. However, since we're not doing decorators I think we should do the following:

1. If we know for sure that the date value can't be nil, then leave the code as it was.
1. If we don't know for sure, then you could use `I18n.l(nil, default: "")`. This will achieve the same effect without adding a new dependency.

(By the way, how I figured out `default: ""` was by looking at the `l` method source code: https://github.com/svenfuchs/i18n/blob/master/lib/i18n/backend/base.rb#L64

Reply:
👍